content/influxdb/cloud-dedicated/reference/cli/influxctl/token/create.md

```diff
@@ -40,48 +40,56 @@ influxctl token create \
 | Flag |                    | Description                           |
 | :--- | :----------------- | :------------------------------------ |
 | `-h` | `--help`           | Output command help                   |
-|      | `--read-database`  | Grant read permissions to a database  |
-|      | `--write-database` | Grant write permissions to a database |
+|      | `--read-database`  | Grant read permissions to a database. Repeatable.  |
+|      | `--write-database` | Grant write permissions to a database. Repeatable. |
 
 ## Examples
 
 - [Create a token with read and write access to a database](#create-a-token-with-read-and-write-access-to-a-database)
 - [Create a token with read-only access to a database](#create-a-token-with-read-only-access-to-a-database)
-- [Create a token with read-only access to multiple database](#create-a-token-with-read-only-access-to-multiple-database)
-- [Create a token with mixed permissions on multiple database](#create-a-token-with-mixed-permissions-on-multiple-database)
+- [Create a token with read-only access to multiple databases](#create-a-token-with-read-only-access-to-multiple-databases)
+- [Create a token with mixed permissions to multiple databases](#create-a-token-with-mixed-permissions-on-multiple-databases)
 
 ##### Create a token with read and write access to a database
 
+{{% code-placeholders "DATABASE_NAME" %}}
 ```sh
 influxctl token create \
-  --read-database mydb \
-  --write-database mydb \
-  "Read/write token for mydb"
+  --read-database DATABASE_NAME \
+  --write-database DATABASE_NAME \
+  "Read/write token for DATABASE_NAME"
 ```
+{{% /code-placeholders %}}
 
```

Comment:
Replace the following:

- {{% code-placeholder-key %}}`DATABASE_NAME`{{% /code-placeholder-key %}}: your {{% cloud-name %}} database